src/main/java/org/example/App.java

```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+
+public class App 
+{
```

Comment:
Please use open curly braces after the name of the class/method. E.g.

public class App {
...
}

Reply:
Ok, I've changed it